### PR TITLE
Fixes the "Quit", "Logout", and "Return to barrack", buttons.

### PR DESCRIPTION
**Pull request by celophi**

### CODE REVIEW

File: doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// File Mask: 
+//  ID Bytes: 
+//   History: 
+//------------------------------------------------
+
+short op;
+int sequence;
+int checksum;
+
+string destination; // Maximum length is 32 characters.
```
Comment (exectails) [inline doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt]: If this is a fixed size string, shouldn't it be char[32]?

Comment (celophi) [reply doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt]: It's not fixed. If you write it as `char[32]`, the array will contain junk bytes. (The value of the column displayed will be correct though).

Comment (exectails) [reply doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt]: It's not? It's always 32 byte, isn't that the definition of fixed?^^ You're right that the array will contain junk the client didn't clear of course, but what will you do if, for example, they decide to add an int after destination now?

Comment (celophi) [reply doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt]: Oh I see. Perhaps I didn't realize what "fixed" means.

Comment (exectails) [reply doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt]: At least that's what I meant^^

**Pull request merged**